webpack.addon.config.js

```diff
@@ -0,0 +1,26 @@
+"use strict";
+const WebpackNotifierPlugin = require("webpack-notifier");
+const webpack = require("webpack");
+const path = require("path");
+
+module.exports = {
+  entry: {
+    "vendor": path.join(__dirname, "./addon-src/vendor.js")
```

Comment:
Not sure if you want the `./` and `/` in the paths here, or just use the more platform agnostic:
```js
path.join(__dirname, "addon-src", "vendor.js")
```